### DISCUSSION TITLE
Memory limit reached

### DISCUSSION THREAD

**Discussion by johns0808** [nftchef/art-engine#264] [Help + Q&A]
Hello,

I am trying to generate 7000 images on a ubuntu server with 40gb of ram but it failed after some times with the error "Memory limit reached..."

I also tried to increase the limit of nodejs with `node --max-old-space-size=20500` but still the same.

Is there a way to increase the memory or re-run the process without restarting from scratch?

**Comment by advra** [comment]
Can you post your limit configs? Run `vi /etc/security/limits.conf`

**Comment by johns0808** [comment]
@advra Everything is commented on this file (prefixed by #) so it's basically empty

**Comment by nftchef** [comment]
@johns0808 not sure what the memory issue is, however, yes, you can split your run up into multiple runs (though it may still use a bit of memory holding the DNA used in memory)

after running the script once, back up the build directory (rename it to build-1 for example)
Change the `startIndex` in config.js

Then run the generator using  the --continue flag, passing in the DNA file path

```
node index.js generate --continue  ./build-1/_dna.json
``` 

converting this to a discussion until an error can be identified 🙇
